app/js/views/app_view.js

```diff
@@ -1,20 +1,21 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'models/todo',
+  'collections/todos',
+  'views/add_todo_view',
+  'views/todos_view'
+], function ($, Backbone, ToDo, ToDoCollection, AddToDoView, ToDosView) {
   return Backbone.View.extend({
-    el: this.$('#todo-app'),
-
-    currentDate: function () {
-      return new Date();
+    initialize: function () {
+      this.toDosView = new ToDosView({model: new ToDoCollection(), el: this.$('#todo-list')});
+      this.addToDoView = new AddToDoView({model: this.toDosView, el: this.$('#todo-input')});
```

Comment:
Generally, the `model` option of a view is populated with an instance of `Backbone.Model`